### PR TITLE
Fix #45 - ability to clear multiple headings

### LINKED ISSUE #45: Support clearing from multiple headings
You can't do the reverse of setting a heading style, which is clearing numbering from multiple headings

### PR DESCRIPTION
**Pull request by caffodian**
refactor + add support to be able to clear numbering from all headings in the current selection

### CODE REVIEW

**Comment by caffodian** [comment]
This needs #44

File: structuredheadings/plugin.js [hunk 1]
```diff
@@ -294,41 +294,47 @@
...
-            this.setValue(value, value);
           }
-        },
 
-        setAutonumberClassesForHeading: function (heading, value) {
-          setNumbering(editor, heading);
-          setLevel(editor, heading);
-          setCurrentStyle(editor, heading, value);
+          // apply the correct bulletstyle for all numbered headings
+          editor.execCommand("reapplyStyle", value);
```
Comment (bocribbz) [inline structuredheadings/plugin.js:334]: is `reapplyStyle` tested somewhere else?

Comment (caffodian) [reply structuredheadings/plugin.js:334]: It is indirectly tested.  It's some of the leftover code i haven't touched yet.

It goes through all the autonumbered headings and basically ensures that the currently selected bullet type is re-applied to all of them (you can only have one autonumbering type at a time currently.).

So any of the multi-heading related tests indirectly test it

**Review by bocribbz — APPROVED** [review]
Looks great 👍

**Pull request merged**